### PR TITLE
Erick

### PR DESCRIPTION
**Pull request by ndurumo254**
updated matrix_power function

### CODE REVIEW

**Comment by ndurumo254** [comment]
updated matrix_power

File: ivy/functional/frontends/jax/lax/linalg.py [hunk 1]
```diff
@@ -10,3 +10,17 @@ def symmetrize(x):
         x = symmetrize(x)
 
     return ivy.cholesky(x)
+
```
Comment (simonetgordon) [inline ivy/functional/frontends/jax/lax/linalg.py:13]: Too many blank lines. There should just be 2.

File: ivy/functional/frontends/jax/lax/linalg.py [hunk 2]
```diff
@@ -10,3 +10,17 @@ def symmetrize(x):
...
+
+
+
+
+def matrix_power(m: Union[ivy.Array, ivy.NativeArray],
```
Comment (simonetgordon) [inline ivy/functional/frontends/jax/lax/linalg.py:18]: There should not be any type hints in the frontend functions. Delete the type-hint related imports too.

File: ivy/functional/frontends/jax/lax/linalg.py [hunk 3]
```diff
@@ -10,3 +10,17 @@ def symmetrize(x):
...
+                 n: int,
```
Comment (simonetgordon) [inline ivy/functional/frontends/jax/lax/linalg.py:19]: Change first two function args to `(a, n, `

**Review by simonetgordon — CHANGES_REQUESTED** [review]

**Comment by simonetgordon** [comment]
Thanks for the contribution! You can check out some of the review comments. We'll also need you to delete changes in the .idea files before merging.

**Pull request closed without merging**